artifacts/manifests/600_mutating.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: clusterresourceoverrides.admission.autoscaling.openshift.io
   labels:
     clusterresourceoverride: "true"
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
```

Comment:
Since we're dropping all the cert generation scripts, we're making the assumption here that going forward this operand will _always_ be used on OpenShift (or at least somewhere where we have a cert injector available), correct? 

(Like I don't think there is anyone out there using the CRO off-label, just want to make sure we're acknowledging we're explicitly choosing that they can't now without some extra work of their own?)

Reply:
Yeah, I think a few months ago, Joel said we were assuming no one is going to use this off OpenShift. I can't find wherever he said it anymore (whether in Slack, GitHub, or over call), but I don't want to quote him if I am wrong so... @joelsmith 

Maybe do we want to make this apparent in the README or something?